### PR TITLE
Multiple maintenance work + adding Farhan as a CODEOWNER

### PR DESCRIPTION
**Pull request by v0lkan**
* Add favicons for the new VSecM pictogram.
* Update roadmap, add new release (content still TBD since our current release will change too)
* Update maintainers.md
* Update CODEOWNERS
* Add a “notification” section to notify last minute changes on the website.
* Add disclaimer to blog that the content belongs to their owners and not VMware
* Other minor changes.

### CODE REVIEW

File: CODEOWNERS
Comment (v0lkan) [inline CODEOWNERS:17]: After merging this in; I’ll have to give Farhan write access to the repo.

Otherwise, it will remain invalid.

File: docs/_includes/head.html
Comment (v0lkan) [inline docs/_includes/head.html:41]: using the new pictogram for favicons too for consistency.

File: docs/_includes/notification.html
Comment (v0lkan) [inline docs/_includes/notification.html:1]: it’s messy, but it gets the job done for now.

we can clean it up later.

File: docs/_pages/0280-roadmap.md
Comment (v0lkan) [inline docs/_pages/0280-roadmap.md:39]: Moved closed iterations to the end of the page.

File: docs/_pages/0003-maintainers.md
Comment (farhan-pasha) [reply docs/_pages/0003-maintainers.md:60]: Thank you @v0lkan 🙇‍♂️

**Review by abhishek44sharma — APPROVED** [review]
lgtm

**Pull request merged**
